### PR TITLE
chore(form): add unit test serializedValue

### LINKED ISSUE #838: [form] setting serialized value on form throws error
Setting serializedValue on lion-form throws an error in the code that is setting the serialized value on the field.

`<lion-form name="test" id="test" .serializedValue=${{firstName: 'Foo'}}>
<form>
<lion-input name="firstName" id="firstName"></lion-input>
</form>
</lion-form>`

Unhandled rejection: TypeError: Cannot set property 'serializedValue' of undefined
   at http://localhost:9648/packages/form-core/src/form-group/FormGroupMixin.js:339:47

### PR DESCRIPTION
**Pull request by T-Motion**
closes #838

### CODE REVIEW

File: packages/form/test/lion-form.test.js [hunk 1]
```diff
@@ -180,4 +181,40 @@ describe('<lion-form>', () => {
...
+ 
+    const el = await fixture(html`
+      <lion-form name="test" id="test" .serializedValue=${{firstName: 'Foo'}}>
+        <form>
+          <lion-input name="firstName" id="firstName"></lion-input>
+        </form>
+      </lion-form>
+    `);
+
+    await el.updateComplete;
```
Comment (daKmoR) [inline packages/form/test/lion-form.test.js:214]:
```suggestion
    await el.registrationComplete;
```

**Comment by T-Motion** [comment]
#838

**Comment by daKmoR** [comment]
@T-Motion hey, so the problem was a missing import to `lion-input.js` 😅  besides that everything works as expected and all tests are green 💪

**Comment by daKmoR** [comment]
I'm going to close as it seem to be not an issue

feel free to break the test again and reopen if needed 🤗

**Pull request closed without merging**